api/models/fixtures/agent_label.json

```diff
@@ -75,6 +75,42 @@
         },
         "Yes"
       ]
+    },
+    "RegionHasCustomTopicUpdate": {
```

Comment:
Just looked at rack.json. Should add a `LambdaRuntime` attribute next to `Ami` in `RegionConfig`